.gitlab-ci.yml

```diff
@@ -0,0 +1,72 @@
+stages:
+  - init
+  - rpmbuild
+  - test
+  - finish
+
+.terraform:
+  after_script:
+    - schutzbot/update_github_status.sh update
+    - schutzbot/run_cloud_cleaner.sh
```

Comment:
`after_script` is overwritten in all jobs derived from `.terraform`, let's just remove this block.